core/src/main/resources/hudson/Messages_fr.properties

```diff
@@ -109,6 +109,7 @@ ProxyConfiguration.TestUrlRequired=Une URL de test est requise.
 ProxyConfiguration.MalformedTestUrl=L''URL de test {0} n''est pas correctement formée.
 ProxyConfiguration.FailedToConnectViaProxy=Impossible de se connecter à {0}.
 ProxyConfiguration.FailedToConnect=Impossible de se connecter à {0} (code {1}).
+ProxyConfiguration.NonTLSWarning=Veuillez noter que cette valeur pourrait être exposée, car la connexion au proxy n'utilise pas TLS
```

Comment:
Are you sure the single quote is OK here? The other values in the file have two single quotes.
(I don't remember the rules for the encoding there, just a warning ;-) )

Reply:
Yes, you are right, sorry. Fixed in 203e3b2ebbaf6adfdf6abb3173c3c2a1a8d327e5